recipes/bioconductor-monocle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = 'monocle' %}
+{% set version = '2.4.0' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://bioconductor.org/packages/release/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz
+  md5: fef48c57d5da92fae71cb501d8603cf0
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
+    - r-biobase
```

Comment:
Yes, I believe that is correct. 

I made this recipe by modifying a similar recipe for another R tool. I tried following that guide without success. I also found this resource (https://github.com/bioconda/bioconda-recipes/tree/master/scripts/bioconductor) and was also unsuccessful at executing the commands without error.